rust/agents/scraper/migration/src/m20220805_000001_create_table_domain.rs

```diff
@@ -31,6 +31,14 @@ const DOMAINS: &[RawDomain] = &[
         is_test_net: false,
         is_deprecated: false,
     },
+    RawDomain {
+        name: "arbitrumgoerli",
+        token: "AGOR",
```

Comment:
Not ETH?

Reply:
I am going off of chainlist.org and that is the currency according to them.

Reply:
No complaints here, changed.